specification/compute/resource-manager/readme.md

```diff
@@ -219,8 +219,7 @@ input-file:
 - Microsoft.Compute/stable/2020-06-01/runCommands.json
 - Microsoft.Compute/stable/2019-04-01/skus.json
 - Microsoft.Compute/stable/2020-06-30/disk.json
-- Microsoft.Compute/preview/2020-09-30/gallery.json
-- Microsoft.Compute/preview/2020-09-30/sharedGallery.json
+- Microsoft.Compute/stable/2019-12-01/gallery.json
```

Comment:
@ArcturusZhang @weidongxu-microsoft @msyyc @qiaozha to comments the sdk impact for this gallery api change

Reply:
Compute for python is multiapi, so it is not permitted to delete tag because there may be customers to use the tag now. It's ok to add a new tag.

Reply:
We have an ongoing email thread to discuss this impact. This is to remove breaking changes that were introduced in the gallery version.

Reply:
per email discussion, Go, Java isn't released, but Python released.

Reply:
Per email thread with Gallery team, no end users are whitelisted to use the 2020-09-30 version of Gallery and SharedGallery. No end behaviour should change. 
@JeffreyRichter approved these changes, so I think we can proceed.